tutorials/scripting/gdextension/gdextension_docs_system.rst

```diff
@@ -0,0 +1,172 @@
+.. _doc_gdextension_docs_system:
+
+GDExtension documentation system
+================================
+
+.. note::
+
+    Adding documentation for GDExtensions is only possible for 4.3 and later. The support can be integrated into your project
```

Comment:
```suggestion
    Adding documentation for GDExtensions is only possible for Godot 4.3 and later. The support can be integrated into your project
```

This may seem obvious but may still be worth clarifying once at the beginning.